sig-autoscaling/annual-report-2021.md

```diff
@@ -0,0 +1,69 @@
+
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being
+retired? Are OWNERS.md files up to date in these areas?
+  * The SIG maintains direct ownership of all of the listed subprojects, with all discussed in one weekly meeting.
+  * Updates for cloud provider implementations for the cluster autoscaler are delegated to the owners of these, with work cutting across these communicated both through SIG meetings and asynchronously through Github Issues/PR mentions.
+
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+  * 2020/08/20 - [Recording here](https://youtu.be/oDL3Kp5-9eM?t=776)
+
+## Membership
+
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  * Yes - though some owners of cloud provider implementations for CA may no longer be active.
+  * Currently no proactive process for pruning inactive members from OWNERs files.
+
+* How do you measure membership? By mailing list members, OWNERs, or something else?
+  * Active participants in SIG meetings and pull requests.
+
+* How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  * No official process for measuring reviewer and approver bandwidth, stuck or controversial PRs are usually discussed at weekly Office Hours.
+  * Reviewer bandwidth is occasionally a restricting factor as we have a small group of approvers, with a small number of experts in each of the SIGs three main areas of focus (Horizontal Pod Autoscaling, Cluster Autoscaler, Vertical Pod Autoscaler).
+
+* Is there a healthy onboarding and growth path for contributors in your SIG?
+What are some activities that the group does to encourage this? What programs are you participating in to grow contributors
+throughout the contributor ladder?
+  * We currently don't actively participate in any programs to grow contributors through the contributor ladder.
+  * We do implement and encourage a [lighter weight proposals](https://github.com/kubernetes/autoscaler/pull/3914) process for the autoscaler repository to allow new users to propose functionality in a structured reviewable manner without the full weight of the KEP process.
+
+* What programs do you participate in for new contributors?
+  * As above, currently none.
+
+* Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that
+they currently are not?
+  * Yes, members of a number of companies (both vendor and end user) regularly contribute to all of the projects owned by the SIG, especially with Cluster Autoscaler's cloud provider implementations. The variety of approvers on core code however is significantly more narrow, with the majority of approvers across the three projects being from a single company.
```

Comment:
is there a pipeline of folks that have been identified or expressed an interest to help in this area?

Reply:
It varies depending on the area. 

There's a number of engaged contributors in the VPA space helping to drive this forward.

The cluster autoscaler generally has engaged cloud provider owners, but the core code is generally contributed to by fewer people with fewer current obvious candidates to progress through the contributor ladder.

The HPA has a number of developers who have previously contributed large areas of new functionality, or who are currently expressing interest in helping to graduate features currently in alpha or beta. As we're currently putting together the plan for finally graduating the v2 APIs from beta, this presents a natural opportunity to try to mentor a number of contributors who have expressed interest in this work.